docs/sources/http_api/data_source.md

```diff
@@ -604,13 +604,108 @@ Content-Type: application/json
 
 Proxies all calls to the actual data source.
 
-## Query a data source by ID
+## Query a data source
 
-Queries a data source having backend implementation.
+Queries a data source having a backend implementation.
+
+`POST /api/ds/query`
+
+> **Note:** Most of Grafana's builtin data sources have a backend implementation.
+
+**Example request for the Test data source**:
+
+```http
+POST /api/ds/query HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+
+{
+   "queries":[
+      {
+         "refId":"A",
+         "scenarioId":"csv_metric_values",
+         "datasource":{
+            "uid":"PD8C576611E62080A"
+         },
+         "format": "table"
+         "maxDataPoints":1848,
+         "intervalMs":200,
+         "stringInput":"1,20,90,30,5,0",
+      }
+   ],
+   "from":"now-5m",
+   "to":"now"
+}
+```
+
+> **Note:** The `from`, `to`, and `queries` properties are required.
```

Comment:
You do not need a note here. You can simply label the "from/to" and "queries" properties as required in the section below.